sass/mixins/_remify.scss

```diff
@@ -0,0 +1,32 @@
+// Defaults
+$base_font_size: 16px;
+$rem_fallback: true;
+
+// Usage: rem(16px);
+@function rem(
+  $size,
+  $base_font_size: $base_font_size
+) {
+  @return #{$size / $base_font_size}rem;
+}
+
+// Usage: em(16px);
+@function em(
+  $size,
+  $base_font_size: $base_font_size
+) {
+  @return #{$size / $base_font_size}em;
+}
+
+// Usage: @include remify(font-size, 16px);
+@mixin remify(
```

Comment:
remify wird gar nicht benutzt, oder?

Reply:
Nö, nur rem. Ich wollte trotzdem das ganze Mixin reinkopieren, dann hätte man remify zur Verfügung. Ich denke mir für unsere Browserunterstützung reicht rem aus. Remify könnten wir auch in unseren Variables-Files nicht benutzen, sondern müssten das umbauen und direkt an den jeweiligen Selektoren nutzen.

Reply:
Würde ich wegschmeissen, hat keine Zukunft. 😀

Reply:
✅